### ISSUE TITLE
Device appears but no notification for incoming image

### ISSUE THREAD

**Issue by KloudAlpha** [grishka/NearDrop#76]
Trying to share a image file from Pixel to MacOS Sonoma. Device shows up, but the app never prompts for the PIN displayed when sharing an image. The request eventually times out.

**Comment by grishka** [comment]
Do you have "do not disturb" enabled on macOS?

**Comment by KloudAlpha** [comment]
> Do you have "do not disturb" enabled on macOS?

no, but I was able to find the notification in the side menu thing. not sure why they're not popping up. thanks.

**Comment by PranavRayudu** [comment]
Bumb. I have NearDrop notifications set to Alert but still have the same problem.

**Comment by Hrishi21g** [comment]
@grishka thanks brother... I had DND on in my mac... That's why I didn't get any notification later on... now it's fine.

**Comment by spaceo** [comment]
I have the same problem in Sonoma 14.0. No notification in OSX and my android phones says it failed transfering. However my mac was discovarable from my android phone.

**Comment by MrModest** [comment]
For those who don't see the notification, try to go to Focus settings in your Mac and add "NearDrop" as allowed app for all your focus modes. It helped to me.
